composer.json

```diff
@@ -8,41 +8,46 @@
   "repositories": [
     {
       "type": "composer",
-      "url": "https://packagist.drupal-composer.org"
+      "url": "https://packages.drupal.org/8"
     }
   ],
   "require-dev": {
+    "behat/behat": "^3.0",
     "behat/mink": "~1.7",
     "behat/mink-zombie-driver": "~1.2",
     "behat/mink-goutte-driver": "~1.2",
-    "drupal/drupal-extension": "~3.0",
+    "drupal/drupal-extension": "~3.2",
     "jcalderonzumba/gastonjs": "~1.0.2",
     "jcalderonzumba/mink-phantomjs-driver": "~0.3.1",
     "mikey179/vfsStream": "~1.2",
+    "se/selenium-server-standalone": "^2.53",
     "phpunit/phpunit": "~4.8",
     "drush/drush": "^8",
     "drupal/console": "~1.0.0",
     "phpmd/phpmd": "~2.1",
-    "drupal/admin_toolbar": "~8.1",
+    "drupal/admin_toolbar": "^1.18",
     "drupal/coder": "^8.2",
-    "drupal/devel": "dev-8.x-1.x",
-    "drupal/masquerade": "8.1.0-beta1"
+    "squizlabs/php_codesniffer": "2.*",
+    "drupal/devel": "^1.0.0-rc1",
+    "drupal/masquerade": "2.0.0-beta1",
+    "symfony/css-selector": "~2.8"
   },
   "require": {
     "roave/security-advisories": "dev-master",
     "composer/installers": "^1.0",
     "drupal-composer/drupal-scaffold": "^2.0.0",
-    "cweagans/composer-patches": "1.5.0",
-    "drupal/lightning": "~8.1",
-    "drupal/diff": "~8.1",
-    "drupal/field_group": "~8.1",
-    "drupal/paragraphs": "dev-8.x-1.x",
-    "drupal/restui": "~8.1",
-    "drupal/search_api": "~8.1",
-    "drupal/search_api_solr": "dev-8.x-1.x",
-    "drupal/search_api_page": "~8.1",
-    "drupal/facets": "dev-8.x-1.x",
-    "drupal/components": "~8.1"
+    "cweagans/composer-patches": "~1.6",
+    "acquia/lightning": "~2.0.0",
+    "drupal/core": "~8.2.0",
```

Comment:
You shouldn't need to specify "drupal/core" here. Composer *should* take the dependency given in the Lightning distro itself.